Sources/OpenAPIServiceModel/CreateOpenAPIServiceModel.swift

```diff
@@ -0,0 +1,382 @@
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("*.\(headerName)") {
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("\(operationName).\(headerName)") {
+            return true
+        }
+        
+        if ignoreRequestHeaders.contains("\(operationName).*") {
+            return true
+        }
+        
+        return false
+    }
+    
+    static func getFixedFieldsOperationMembers(fixedFields: OpenAPI.Parameter, operationName: String,
+                                               index: Int, members: inout OpenAPIServiceModel.OperationInputMembers,
+                                               items: JSONSchema, model: inout OpenAPIServiceModel, modelOverride: ModelOverride?) {
+        let typeName = fixedFields.name.safeModelName().startingWithUppercase
+        
+        let fieldName = "\(operationName)Request\(typeName)"
+        let member = Member(value: fieldName,
+                            position: index,
+                            required: fixedFields.required,
+                            documentation: fixedFields.description)
+        switch fixedFields.location {
+        case .query:
+            members.queryMembers[fixedFields.name] = member
+        case .path:
+            members.pathMembers[fixedFields.name] = member
+        case .header:
+            guard !ignoreRequestHeader(operationName: operationName, headerName: fixedFields.name, modelOverride: modelOverride) else {
+                return
+            }
+            
+            members.additionalHeaderMembers[fixedFields.name] = member
+        default:
+            break
+        }
+        
+        addField(item: items, fieldName: fieldName,
+                 model: &model, modelOverride: modelOverride)
+    }
+    
+    static func addOperationResponseFromSchema(schema: JSONSchema, operationName: String, forCode code: Int, index: Int?,
+                                               description: inout OperationDescription,
+                                               model: inout OpenAPIServiceModel, modelOverride: ModelOverride?, document: OpenAPI.Document) {
+        switch schema {
+        case .object:
+            let indexString = index?.description ?? ""
+            var structureName = "\(operationName)\(code)Response\(indexString)Body"
+            var structureDescription = StructureDescription()
+            guard let objectContext = schema.objectContext else {
+                fatalError("Not object")
+            }
+            parseObjectSchema(structureDescription: &structureDescription, enclosingEntityName: &structureName,
+                              model: &model, objectContext: objectContext, modelOverride: modelOverride, document: document)
+            
+            model.structureDescriptions[structureName] = structureDescription
+            
+            if code >= 200 && code < 300 {
```

Comment:
nit: we can define a swift range here: https://www.programiz.com/swift-programming/ranges#:~:text=In%20Swift%2C%20a%20range%20is,...%20is%20a%20range%20operator
and use a variable for the range and define the variable here (and elsewhere)

Reply:
something like`200...299 ~= code` ?

Reply:
`let nonErrorCodeRange = 200...299
...
if nonErrorCodeRange.contains(code)`

I was thinking something like this.

Reply:
Done.